README.md

```diff
@@ -1,11 +1,87 @@
 # Topiary
 
-![CI Status](https://github.com/tweag/topiary/actions/workflows/ci.yml/badge.svg)
+![CI Status][ci-badge]
 
 Topiary aims to be a uniform formatter for simple languages, as part of
-the [Tree-sitter](https://tree-sitter.github.io/tree-sitter/) ecosystem.
-It is named after the art of clipping or trimming trees into fantastic
-shapes.
+the [Tree-sitter] ecosystem. It is named after the art of clipping or
+trimming trees into fantastic shapes.
+
+## Motivation
+
+The style in which code is written has, historically, been mostly left
+to personal choice. Of course, this is subjective by definition and has
+led to many wasted hours reviewing formatting choices, rather than the
+code itself. Prescribed style guides were an early solution to this,
+spawning tools that lint a developer's formatting and ultimately leading
+to automatic formatters. The latter were popularised by
+[`gofmt`][gofmt], whose developers had the insight that "good enough"
```

Comment:
Maybe we ought to link to the slides which talk about design decisions somewhere.

For the record, I don't care too much about the absence of configurability (in fact Topiary is infinitely configurable: just clone the query file, and do the modifications that you want). It is important in Ormolu only because Haskell is so hard to parse and format that it's quite hard to test with configuration options (yes, changing from 2 to 4 spaces of indentation can conceivably break a Haskell output if not done carefully) but here, it doesn't matter too much.

Reply:
Added in fd18ebc1c1a